lib/copy_peste/modules/graphics.rb

```diff
@@ -0,0 +1,52 @@
+require_relative '../modules.rb'
+
+module CopyPeste
+  module Modules
+
+    class Graphics < CopyPeste::Modules::Base
+
+      def initialize
+        super()
+        @alive = true
+        @type = "graphic"
+      end
+
+      # Return the alive boolean variable that define if the CopyPeste Framework
+      # is still running or not.
+      # @param [Boolean] True if CopyPeste Framework is running otherwise False.
+      def running?
```

Comment:
This comment has to be applied to all the other functions of this file. If I understand, this class is the base class of a graphical module. Shouldn't these functions return or raise an error ? Or at least mandatory functions ? To avoid forgetting to overload some of them ?